java/api/src/main/java/org/ray/api/ActorCall.java

```diff
@@ -651,4 +651,37 @@ default void call(RayFuncVoid1<A> f) {
     Ray.internal().callActor(f, (RayActor) this, args);
   }
 
+  // ===========================
+  // Cross-language methods.
+  // ===========================
+  default RayObject call(String functionName) {
```

Comment:
I think we can put this methods in a different interface (e.g., `PyActorCall`) and only let `RayPyActor` inherits `PyActorCall`.

Reply:
I moved Python actor call to PyActorCall.java, and RayPyActor extends PyActorCall.